### PR TITLE
Update Storybook addon metadata

### PR DESCRIPTION
**Pull request by coderkevin**
Hey naver! We've created a catalog of Storybook addons and yours is included: https://storybook.js.org/addons/storybook-addon-preview

To make your addon more discoverable in the catalog, we've curated some of its metadata, and I've included those edits in this PR.

To learn more about the catalog metadata, check out our documentation: https://storybook.js.org/docs/react/addons/addon-catalog

### CODE REVIEW

**Comment by daybrush** [comment]
@coderkevin 

These are good keywords.
However, it seems that the indent has changed to 2. In this case, it is used as indent 4.

**Review by daybrush — APPROVED** [review]

**Review by WoodNeck — APPROVED** [review]
Thanks for PR! 🤗 

@daybrush 
Maybe we should update the image src in the README. It's broken in storybook's catalog.

File: package.json [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
-        "typescript": "^3.7.2"
-    }
+  "name": "storybook-addon-preview",
+  "version": "2.1.1",
+  "description": "Storybook Addon Preview can show user selected knobs in various framework code in Storybook",
+  "main": "./dist/index.js",
+  "module": "./dist/esm/index.js",
+  "declaration": "./dist/index.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (mixed) [inline package.json:9]: Regardless of this PR, it seems to be the old code. Since this is unnecessary code, it would be nice to delete it this time.

**Review by mixed — APPROVED** [review]
LGTM!

**Comment by coderkevin** [comment]
Thanks for the edits, cheers!

**Pull request merged**